internal/pkg/service/stream/storage/level/local/diskwriter/network/rpc/pb/networkFile.proto

```diff
@@ -14,7 +14,8 @@ service NetworkFile {
 }
 
 message OpenRequest {
-  SliceKey slice_key = 1;
+  string source_node_id = 1;
+  SliceKey slice_key = 2;
 }
 
```

Comment:
Source node reports its own node ID to the disk writer node, on network file open.